test/integration/users.test.js

```diff
@@ -1229,4 +1230,71 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("test discord actions for nickname for verified user", function () {
+    beforeEach(async function () {
+      fetchStub = Sinon.stub(global, "fetch");
+      userId = await addUser(userData[0]);
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+      Sinon.restore();
+    });
+
+    it("returns 200 for updating nickname patch method", function (done) {
+      fetchStub.returns(
+        Promise.resolve({
+          status: 200,
+          json: () => Promise.resolve({}),
+        })
+      );
+      chai
+        .request(app)
+        .patch(`/users/servername/${userId}`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
```

Comment:
the route isn't `SUPERUSER` restricted, I guess we do not need to provide `SuperUserAuthToken` for this

Reply:
in the current iteration, it is super user restricted.